dds_web/api/user.py

```diff
@@ -165,30 +183,44 @@ def add_user_to_project(existing_user, project, role):
             }
 
         owner = role == "Project Owner"
-
-        project = project_schemas.ProjectRequiredSchema().load({"project": project})
         ownership_change = False
-        for rusers in project.researchusers:
-            if rusers.researchuser is existing_user:
+
+        if isinstance(whom, models.ResearchUser):
+            link = project.researchusers
+        else:
+            link = project.invites
+
+        for rusers in link:
+            if rusers.researchuser is whom:
```

Comment:
I think it would be safer to use `==` instead of `is` here. `is` checks that the two variables actually reference the exact same object, so I'm not sure if two different instances of the same user would be evaluating to true here.

Reply:
Sure, but this is essentially code from @inaod568, so I kept the `is` as it is now:  
https://github.com/ScilifelabDataCentre/dds_web/blob/b573bec26d44a3fd159950d638025e2b1d453086/dds_web/api/user.py#L171-L181